Mathlib/Data/Set/Lattice.lean

```diff
@@ -1363,6 +1363,9 @@ theorem biUnion_iUnion (s : ι → Set α) (t : α → Set β) :
     ⋃ x ∈ ⋃ i, s i, t x = ⋃ (i) (x ∈ s i), t x := by simp [@iUnion_comm _ ι]
 #align set.bUnion_Union Set.biUnion_iUnion
 
+lemma iUnion_biUnion {ι κ : Type*} (s : ι → Set α) {j : κ → Set ι} :
```

Comment:
Besides the main mathematical result about metrizability of convergence in distribution, this PR suggests adding some number of `biUnion` lemmas (all of which were counterparts of some existing `iUnion` lemmas) that in the end are not strictly necessary for the main proof, but I expected to find them and thought they could be seen as API-holes.

The lowest level such lemma is `iUnion_biUnion` here. This is one thing I expected to find by `exact?`.

I'm ok leaving these `biUnion` lemmas out of the PR if they are not seen as natural API additions. I'll repeat the comment briefly at each such lemma.